loom/vendor/stb/stb_truetype.h

```diff
@@ -630,7 +630,7 @@ enum { // languageID for STBTT_PLATFORM_ID_MAC
 static int stbtt__isfont(const stbtt_uint8 *font)
 {
    // check the version number
-   if (stbtt_tag(font, "1"))   return 1; // TrueType 1
+   if (font[0] == '1')            return 1; // TrueType 1
```

Comment:
Yeah, thanks @nothings! We saw that there is a current release that fixes this. I will update to it when I get the chance.